ctapipe_io_nectarcam/__init__.py

```diff
@@ -289,7 +342,7 @@ def unpack_feb_data(self, event):
         # Unpack
         unpacked_feb =  struct.unpack(rec_fmt, event.nectarcam.counters)
```

Comment:
I assume this is similar to what LST has. If yes, we got a good speedup by using numpy, not struct, see: 

* https://github.com/cta-observatory/ctapipe_io_lst/blob/e5e8939a8bae4dd88409181ac714f8b2bea9a86b/ctapipe_io_lst/__init__.py#L507

and the definitions of the data types:

* https://github.com/cta-observatory/ctapipe_io_lst/blob/master/ctapipe_io_lst/anyarray_dtypes.py

Reply:
Okay, we should have another issue about this point.